Documentation/building/windows/instructions.md

```diff
@@ -18,21 +18,15 @@ MSBuild version 15 or later is required.
 
  4. (Optional) [Configure the build](../configuration.md).
 
- 5. (Optional) In a [Developer Command Prompt][developer-prompt], prepare external git dependencies:
-
-        msbuild Xamarin.Android.sln /t:PrepareExternal
-
-    This will configure external proprietary components such as monodroid.
```

Comment:
I don't know if we need to remove these docs here, we still have these targets: https://github.com/xamarin/xamarin-android/blob/8cd3c1544c1f404d0c72d11462a493abece60d1c/build-tools/scripts/PrepareWindows.targets#L14-L17

I usually run `/t:PrepareExternal` to get `monodroid` on my machine.

We have the `Makefile` equivalent, too: https://github.com/xamarin/xamarin-android/blob/8cd3c1544c1f404d0c72d11462a493abece60d1c/Makefile#L238-L239

Reply:
I agree that we shouldn't remove this section of the docs.